.jshintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "quotmark": "single",
+  "node": true,
+  "globals": {
+    "angular": false,
+    "document": false,
+    "jasmine": false,
+    "module": false,
+    "describe": false,
+    "beforeEach": false,
+    "afterEach": false,
+    "it": false,
+    "inject": false,
+    "expect": false,
+    "spyOn": false
+  }
+}
```

Comment:
IMHO, I personally love to separate a file into serval files, if it's reasonable,  and keep each file as small chunk of codes as possible. I think that it's easier to maintain and to work separately on a team. 

In this case, if we wanted to update the option of the jshintrc, we could go changing directly on the `.jshinrc` file instead of `Gruntfile.js`. Moreover, in the future, if we wanted to swap to `gulp`, or any other task runners, we wouldn't have to set up the jshintrc option again.